ocs_ci/utility/rosa.py

```diff
@@ -47,11 +47,17 @@ def create_cluster(cluster_name, version):
     compute_nodes = config.ENV_DATA["worker_replicas"]
     compute_machine_type = config.ENV_DATA["worker_instance_type"]
     multi_az = "--multi-az " if config.ENV_DATA.get("multi_availability_zones") else ""
+    cluster_type = config.ENV_DATA.get("cluster_type", "")
+    provider_name = config.ENV_DATA.get("provider_name", "")
     cmd = (
         f"rosa create cluster --cluster-name {cluster_name} --region {region} "
         f"--compute-nodes {compute_nodes} --mode auto --compute-machine-type "
         f"{compute_machine_type}  --version {rosa_ocp_version} {multi_az}--sts --yes"
     )
+    if cluster_type.lower() == "consumer" and config.ENV_DATA.get("provider_name", ""):
+        subnet_id = get_providers_subnet(region, provider_name)
+        cmd = f"{cmd} --subnet-ids {subnet_id}"
```

Comment:
If the `subnet_id` is an empty string, will the command still work?

Reply:
it comes to this loop only and only if we are using `consumer` config and defining a non empty string for provider name. 
And if we do not define subnet_id to date the job shows rosa error for the subnet_id parameter. 
For now get_providers_subnet() definition is not completed and using hardcoded value from config. In actual, future PR, pending todos of this function will take care of subnet ids